src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,74 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationServiceImpl regService;
+
+    @BeforeAll
+    static void initialize() {
+        regService = new RegistrationServiceImpl();
+    }
+
+    @Test
+    void loginIsAlreadyInStorage_NotOk() {
+        User user1 = new User("Alice", "123456", 19);
+        User user2 = new User("Alice", "654321", 22);
+        assertNotNull(regService.register(user1));
+        assertNull(regService.register(user2));
+    }
+
+    @Test
+    void userAgeIsLessThanMustBe_NotOk() {
+        User user = new User("Alice", "123456", 16);
+        assertNull(regService.register(user));
+    }
+
+    @Test
+    void userPasswordIsLessThanMustBe_NotOk() {
+        User user = new User("Alice", "12345", 19);
+        assertNull(regService.register(user));
```

Comment:
Here you test never pass. You must use AssertThrow, to check Exception what throwing when you give not correct password